src/oemof_tabular_plugins/wefe/facades/functions.py

```diff
@@ -350,24 +354,39 @@ def potential_evapotranspiration(z, t_air, t_dp, w10, r_n, g, **kwargs):
     r_n *= C_WH_TO_J * C_J_TO_MJ / C_H_TO_D  # W/m² over an hour to MJ/(m²*day)
     g *= C_WH_TO_J * C_J_TO_MJ / C_H_TO_D  # W/m² over an hour to MJ/(m²*day)
     # wind speed at 2m above ground [m/s] (Eq. 47)
-    w2 = w10 * 4.87 / np.log(672.58)
+    c1 = 4.87       # [-]
+    c2 = 672.58     # [-]
+    w2 = w10 * c1 / np.log(c2)
     # atmospheric pressure at elevation z [kPa] (Eq. 7)
-    p = 101.3 * ((293 - 0.0065 * z) / 293) ** 5.26
+    c1 = 101.3      # [kPa]
```

Comment:
here `c1` seems to be the reference atmospheric pressure at sea level, however `c1` doesn't really carry this meaning